Dockerfile.15sp3

```diff
@@ -16,13 +19,13 @@ RUN zypper ar http://download.suse.de/ibs/SUSE/Products/SLE-Module-Basesystem/15
     zypper --non-interactive install git-core ruby-devel make gcc gcc-c++ build wget dmidecode \
       vim osc hwinfo libx86emu3 zypper-migration-plugin sudo awk curl
 
-RUN wget http://username:password@gaffer.suse.de:9999/files/.regcode -O ~/.regcode
-RUN wget http://username:password@gaffer.suse.de:9999/files/.oscrc -O ~/.oscrc
+ADD integration/create-oscrc.sh /tmp/connect/integration/create-oscrc.sh
```

Comment:
Use copy instead of ADD

see [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

Reply:
thanks, good point, I did replace it to `COPY` also in other places